config/FindNDI.cmake

```diff
@@ -77,24 +76,34 @@ endif()
 find_path( NDI_INCLUDE_DIR
   NAMES ndi.h
   HINTS ${NDI_ROOT_DIR}/include/ndi ${NDI_ROOT_DIR}/include
-  PATH_SUFFIXES ndi
-  )
-
+  PATH_SUFFIXES ndi )
 set( NDI_LIBRARY_NAME ndipic ndi)
 
 find_library(NDI_LIBRARY
   NAMES ${NDI_LIBRARY_NAME}
-  PATHS ${NDI_ROOT_DIR}/lib
-  )
+  PATHS ${NDI_ROOT_DIR}/lib )
 # Do we also have debug versions?
 find_library( NDI_LIBRARY_DEBUG
   NAMES ${NDI_LIBRARY_NAME}
   HINTS ${NDI_ROOT_DIR}/lib
-  PATH_SUFFIXES Debug
-)
+  PATH_SUFFIXES Debug )
 set( NDI_INCLUDE_DIRS ${NDI_INCLUDE_DIR} )
 set( NDI_LIBRARIES ${NDI_LIBRARY} )
 
+# Try to find the version.
```

Comment:
@KineticTheory If you add the 06a22dd change from the discussion I will happily approve :)

Reply:
It's already there!

Reply:
Weird -- last night github's web interface was saying something like "This commit is not a member of any PR". I see it now though!